runtime-modules/project-token/src/lib.rs

```diff
@@ -1622,6 +1848,11 @@ impl<T: Config> Module<T> {
         <T as Config>::ModuleId::get().into_sub_account_truncating(Vec::<u8>::new())
     }
 
+    /// Returns the account for the AMM treasury
+    pub fn amm_treasury_account(token_id: T::TokenId) -> T::AccountId {
```

Comment:
This is a new "module" account that was not initialized at genesis. I think we need a runtime migration to give it an existential deposit.